tests/integration/s2n_handshake_test_s_client.py

```diff
@@ -437,26 +459,30 @@ def run_handshake_test(host, port, ssl_version, cipher, fips_mode, no_ticket, us
 
     if ssl_version and ssl_version < cipher_vers:
         return 0
-    
+
     client_cert_str=str(use_client_auth)
-    
+
     if (use_client_auth is not None) and (client_cert_path is not None):
         client_cert_str = cert_path_to_str(client_cert_path)
 
-    ret = try_handshake(host, port, cipher_name, ssl_version, no_ticket=no_ticket, enter_fips_mode=fips_mode, client_auth=use_client_auth, client_cert=client_cert_path, client_key=client_key_path)
-    
+    ret = try_handshake(host, port, cipher_name, ssl_version,
+        no_ticket=no_ticket, enter_fips_mode=fips_mode, client_auth=use_client_auth,
+        client_cert=client_cert_path, client_key=client_key_path,
+        **kwargs)
+
     result_prefix = "Cipher: %-30s ClientCert: %-16s Vers: %-8s ... " % (cipher_name, client_cert_str, S2N_PROTO_VERS_TO_STR[ssl_version])
     print_result(result_prefix, ret)
 
     return ret
 
 
-def handshake_test(host, port, test_ciphers, fips_mode, no_ticket=False, use_client_auth=None, use_client_cert=None, use_client_key=None):
+def handshake_test(host, port, test_ciphers, fips_mode,
+        no_ticket=False, use_client_auth=None, use_client_cert=None, use_client_key=None, **kwargs):
     """
     Basic handshake tests using all valid combinations of supported cipher suites and TLS versions.
     """
     print("\n\tRunning handshake tests:")
-    
+
     failed = False
     for ssl_version in [S2N_TLS10, S2N_TLS11, S2N_TLS12, None]:
```

Comment:
NO TLS13 here? Should it be added based on the TLS1.3 flag?

Reply:
It's not clear here but None refers to not specifying a TLS version, which is client default. in openssl s_client case, no versions means potentially negotiating any version up to TLS 1.3 (which explains the other change to cap server to negotiate a maximum of TLS 1.2).